### ISSUE TITLE
Backend error "No root page found" in combination with heimrichhannot/contao-video-bundle

### ISSUE THREAD

**Issue by Webstylerin** [heimrichhannot/contao-newselement#5]
Hello all,

I have a problem in combination with heimrichhannot/contao-video-bundle and enabled news support.

As soon as I select a video provider in the news, I get a "No root page found" error in the backend when I try to open the article where the news is integrated via contao-newselement. The frontend is working fine.

I tested "file" and "youtube". It doesn't matter if I disable the "add video" option. As long as something other than "-" is selected, I get the error.

If I use a newslist module, everything works fine in the frontend and backend.

Contao version: 4.13.5
heimrichhannot/contao-newselement version: 1.2.0
heimrichhannot/contao-video-bundle Version: 1.0.3
PHP version: 7.4.30

After [1/2] NoRootPageFoundException comes [2/2] NotFoundHttpException. I'm including the stacktrace of the first error.

Do you need anything else from me?

```
Contao\CoreBundle\Exception\NoRootPageFoundException:
No root page found

  at vendor/contao/core-bundle/src/Resources/contao/classes/Frontend.php:319
  at Contao\Frontend::getRootPageFromUrl()
     (vendor/heimrichhannot/contao-video-bundle/src/Generator/VideoGenerator.php:93)
  at HeimrichHannot\VideoBundle\Generator\VideoGenerator->generate(object(FileVideo), object(ModuleNewsElement))
     (vendor/heimrichhannot/contao-video-bundle/src/EventListener/ParseArticlesListener.php:61)
  at HeimrichHannot\VideoBundle\EventListener\ParseArticlesListener->onParseArticles(object(FrontendTemplate), array('id' => '2', 'pid' => '1', 'tstamp' => '1656576451', 'headline' => 'Passione di Famiglia', 'featured' => '', 'alias' => 'passione-di-famiglia', 'author' => '1', 'date' => '1656488160', 'time' => '1656488160', 'pageTitle' => '', 'robots' => '', 'description' => null, 'subheadline' => 'Waurig', 'teaser' => null, 'addImage' => '1', 'overwriteMeta' => '', 'singleSRC' => 'Y$�~�_��v', 'alt' => '', 'imageTitle' => '', 'size' => 'a:3:{i:0;s:0:"";i:1;s:0:"";i:2;s:1:"2";}', 'imagemargin' => 'a:5:{s:6:"bottom";s:0:"";s:4:"left";s:0:"";s:5:"right";s:0:"";s:3:"top";s:0:"";s:4:"unit";s:0:"";}', 'imageUrl' => '', 'fullsize' => '', 'caption' => '', 'floating' => 'above', 'addEnclosure' => '', 'enclosure' => null, 'source' => 'default', 'jumpTo' => '0', 'articleId' => '0', 'url' => '', 'target' => '', 'cssClass' => '', 'noComments' => '', 'published' => '1', 'start' => '', 'stop' => '', 'languageMain' => '0', 'sorting' => '256', 'posterSRC' => null, 'transcriptedVimeo' => '', 'transcriptedYoutube' => '', 'videoAlternativeText' => null, 'videoSubtitles' => null, 'videoSRC' => 'a:1:{i:0;a:2:{s:4:"file";s:16:"Wa$4���_��v";s:10:"mediaQuery";s:0:"";}}', 'videoAutoplay' => '1', 'videoFullsize' => '', 'videoLinkText' => '', 'ytShowInfo' => '', 'ytModestBranding' => '', 'videoShowRelated' => '', 'videoDuration' => '', 'videoLoop' => '', 'videoRemoveControls' => '', 'addPlayButton' => '', 'addPreviewImage' => '', 'videoProvider' => 'file', 'vimeo' => '', 'youtube' => '', 'addVideo' => '1', 'cssID' => ''), object(ModuleNewsElement))
     (vendor/contao/news-bundle/src/Resources/contao/modules/ModuleNews.php:215)
  at Contao\ModuleNews->parseArticle(object(NewsModel))
     (vendor/heimrichhannot/contao-newselement/modules/ModuleNewsElement.php:46)
  at HeimrichHannot\Newselement\ModuleNewsElement->compile()
     (vendor/contao/core-bundle/src/Resources/contao/modules/Module.php:214)
  at Contao\Module->generate()
     (vendor/heimrichhannot/contao-newselement/modules/ModuleNewsElement.php:41)
  at HeimrichHannot\Newselement\ModuleNewsElement->generate()
     (vendor/heimrichhannot/contao-newselement/elements/ContentNewsArticle.php:26)
  at HeimrichHannot\Newselement\ContentNewsArticle->generate()
     (vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:615)
  at Contao\Controller::getContentElement(object(ContentModel))
     (vendor/contao/core-bundle/src/Resources/contao/dca/tl_content.php:1342)
  at tl_content->addCteType(array('id' => '8', 'type' => 'newselement', 'pid' => '1', 'ptable' => 'tl_article', 'sorting' => '640', 'tstamp' => '1656489472', 'headline' => 'a:2:{s:4:"unit";s:2:"h2";s:5:"value";s:0:"";}', 'text' => null, 'addImage' => '', 'showPreview' => '', 'inline' => '', 'overwriteMeta' => '', 'singleSRC' => null, 'alt' => '', 'imageTitle' => '', 'size' => 'a:3:{i:0;s:0:"";i:1;s:0:"";i:2;s:1:"2";}', 'imagemargin' => '', 'imageUrl' => null, 'fullsize' => '', 'caption' => '', 'floating' => 'above', 'html' => null, 'listtype' => '', 'listitems' => null, 'tableitems' => null, 'summary' => '', 'thead' => '', 'tfoot' => '', 'tleft' => '', 'sortable' => '', 'sortIndex' => '0', 'sortOrder' => 'ascending', 'mooHeadline' => '', 'mooStyle' => '', 'mooClasses' => '', 'highlight' => '', 'markdownSource' => 'sourceText', 'code' => null, 'url' => null, 'target' => '', 'overwriteLink' => '', 'titleText' => '', 'linkTitle' => '', 'embed' => '', 'rel' => '', 'useImage' => '', 'multiSRC' => null, 'orderSRC' => null, 'useHomeDir' => '', 'perRow' => '4', 'perPage' => '0', 'numberOfItems' => '0', 'sortBy' => '', 'metaIgnore' => '', 'galleryTpl' => '', 'customTpl' => '', 'playerSRC' => null, 'youtube' => '', 'vimeo' => '', 'posterSRC' => null, 'playerSize' => '', 'playerOptions' => null, 'playerStart' => '0', 'playerStop' => '0', 'playerCaption' => '', 'playerAspect' => '', 'splashImage' => '', 'playerPreload' => '', 'playerColor' => '', 'youtubeOptions' => null, 'vimeoOptions' => null, 'sliderDelay' => '7000', 'sliderSpeed' => '700', 'sliderStartSlide' => '0', 'sliderContinuous' => '', 'data' => null, 'cteAlias' => '0', 'articleAlias' => '0', 'article' => '0', 'form' => '0', 'module' => '0', 'protected' => '', 'groups' => null, 'guests' => '', 'cssID' => 'a:2:{i:0;s:0:"";i:1;s:0:"";}', 'invisible' => '', 'start' => '', 'stop' => '', 'toolbox_classes' => '', 'rsce_data' => null, 'news_categoryImgSize' => '', 'news_module' => '0', 'news_categoryFilterPage' => '0', 'news_filterPreserve' => '0', 'news_filterDefault' => null, 'news_includeSubcategories' => '0', 'news_relatedCategories' => '0', 'news_filterCategories' => '0', 'sliderCustomOptions' => null, 'sliderCenteredSlides' => '', 'sliderAutoheight' => '', 'sliderBreakpoints' => null, 'sliderPaginationType' => 'bullets', 'sliderPagination' => '', 'sliderScrollbar' => '', 'sliderButtons' => '', 'sliderWrapperClass' => '', 'sliderEffect' => 'slide', 'sliderSlidesPerView' => '1', 'sliderSpaceBetween' => '0', 'news_metaFields' => '', 'news_template' => 'news_projekt', 'news' => '2', 'news_archive' => '1', 'transcriptedVimeo' => '', 'transcriptedYoutube' => '', 'videoAlternativeText' => null, 'videoSubtitles' => null, 'videoSRC' => null, 'videoAutoplay' => '', 'videoFullsize' => '', 'videoLinkText' => '', 'ytShowInfo' => '', 'ytModestBranding' => '', 'videoShowRelated' => '', 'videoDuration' => '', 'videoLoop' => '', 'videoRemoveControls' => '', 'addPlayButton' => '', 'addPreviewImage' => '', 'videoProvider' => ''))
     (vendor/contao/core-bundle/src/Resources/contao/drivers/DC_Table.php:4635)
  at Contao\DC_Table->parentView()
     (vendor/contao/core-bundle/src/Resources/contao/drivers/DC_Table.php:312)
  at Contao\DC_Table->showAll()
     (vendor/contao/core-bundle/src/Resources/contao/classes/Backend.php:670)
  at Contao\Backend->getBackendModule('article', null)
     (vendor/contao/core-bundle/src/Resources/contao/controllers/BackendMain.php:168)
  at Contao\BackendMain->run()
     (vendor/contao/core-bundle/src/Controller/BackendController.php:49)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (vendor/symfony/http-kernel/HttpKernel.php:152)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (vendor/symfony/http-kernel/HttpKernel.php:74)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (vendor/symfony/http-kernel/Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (web/index.php:44)
```

Regards, Nicole

**Comment by koertho** [comment]
Hi, that should be fixed in version 1.3.

**Comment by Webstylerin** [comment]
Perfect, thank you.
